### PR TITLE
Engine texture fixes

### PR DESCRIPTION
**Pull request by haxiomic**
- Added missing gl.generateMipmaps to Engine createRawTexture
- Removed gl.generateMipmaps from createRenderTargetTexture since it's generating mipmaps of blank texture (and mipmaps are generated for the framebuffer color attachment elsewhere)

### CODE REVIEW

File: src/babylon.engine.ts [hunk 1]
```diff
@@ -2344,10 +2349,6 @@
             }
             gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0);
 
-            if (generateMipMaps) {
```
Comment (deltakosh) [inline src/babylon.engine.ts]: This may be required to get a complete texture. Because this could be bind as a texture before a render is done

Comment (haxiomic) [inline src/babylon.engine.ts]: You're right, good spot, thanks! Will undo this change

**Comment by nockawa** [comment]
So the mipmap behavior was unchanged for createRenderTargetTexture? Because I do use it on C2D for cached Groups.

**Comment by deltakosh** [comment]
nope did not change. only for rawtexture
